src/web/components/ParticipantManagement/UpdateParticipantDialog.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import { SubmitHandler } from 'react-hook-form';
+
+import { ApiRoleDTO } from '../../../api/entities/ApiRole';
+import { ParticipantDTO } from '../../../api/entities/Participant';
+import { UpdateParticipantForm } from '../../services/participant';
+import { Dialog } from '../Core/Dialog';
+import { Form } from '../Core/Form';
+import { CheckboxInput } from '../Input/CheckboxInput';
+
+type UpdateParticipantDialogProps = {
+  triggerButton: JSX.Element;
+  participant: ParticipantDTO;
+  onUpdateParticipant: (form: UpdateParticipantForm, participant: ParticipantDTO) => Promise<void>;
+  apiRoles: ApiRoleDTO[];
+};
+
+function UpdateParticipantDialog({
```

Comment:
nit: I feel like it's good practice to have the props listed here in the same order as the above type?

Reply:
Ah yeah that makes sense.